### ISSUE TITLE
Keras 2.0.2 - VarianceScaling has no attribute __name__ when using model.to_json

### ISSUE THREAD

**Issue by flybass** [fchollet/keras#6069]
This happens when I utilize the batch normalization layer. 

This is as the title suggests. I've been a keras user since 0.3 and this happened with the recent update to 2 from 1. 

Thanks,
Daniel
